src/scripts/statusbar.ts

```diff
@@ -85,31 +132,34 @@ namespace Scripts {
                 statusBar.hp === hp &&
                 statusBar.max === max &&
                 statusBar.name === name &&
-                statusBar.poisoned === poisoned
+                statusBar.poisoned === poisoned &&  
+                !indicatorsChanged
             ) {
                 return;
             }
 
             let updateText = false;
-
+   
             // dead change state (ressurection)
             // visible ghost (turning warmode on)
-            if (statusBar.dead !== dead || (!statusBar.visible && dead)) {
+            // flags change
```

Comment:
dal bych samostatnou podminku na zacatek pro indicatorsChanged kde by se volalo drawIndicators.. ten polygon je mimo pozici ostatnich polygonu, takze ho neni potreba stale prekreslovat

Reply:
No nevim, 3 stech bloku delaj gump.clear(), takze pokud se toto vykresli na zacatku. V jednom pruchodu muze byt vic splnenych podminek. 
Jako by the book chapu, ale jinak to asi valnyho vyznamu nema.

Reply:
Chapu